tests/com/tony/server/response/MethodNotAllowedResponseTest.java

```diff
@@ -2,13 +2,13 @@
 
 import org.junit.Test;
 
-import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
 
 public class MethodNotAllowedResponseTest {
     @Test
     public void testResponseBuiltWithCorrectResponse() throws Exception {
         MethodNotAllowedResponse methodNotAllowedResponse = new MethodNotAllowedResponse();
-        String expectedResponse = "HTTP/1.1 405 Method Not Allowed\n";
+        String expectedResponse = "HTTP/1.1 405 Method Not Allowed\r\n";
```

Comment:
It's pretty crazy how many places you had to change this, because of a change to your ResponseBuilder, isn't it?

This probably would have been an easier change if you had a test helper to build these test responses for you.